src/analyses/region.ml

```diff
@@ -82,12 +81,8 @@ struct
       Some (Lvals.empty ())
     | Memory {exp = e; _} ->
       (* TODO: remove regions that cannot be reached from the var*)
-      let rec unknown_index = function
-        | `NoOffset -> `NoOffset
-        | `Field (f, os) -> `Field (f, unknown_index os)
-        | `Index (i, os) -> `Index (Lval.any_index_exp, unknown_index os) (* forget specific indices *)
-      in
-      Option.map (Lvals.of_list % List.map (Tuple2.map2 unknown_index)) (get_region ctx e)
+      (* forget specific indices *)
+      Option.map (Lvals.of_list % List.map (Tuple2.map2 Offset.Exp.top_indices)) (get_region ctx e)
```

Comment:
If we forget them here, do we not want to avoid storing them in the first place?

Reply:
Maybe, I just added a TODO about it instead of refactoring the region analysis more deeply.
As noted in #107, region analysis uses `exp`s for some reason, although maybe `lval`s (or now our `mval`s) could actually suffice. In that case one could easily use `Mval.Unit`.